tests/operations.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitAndStartInstance,
+  visitAndStopInstance,
+} from "./helpers/instances";
+import { assertTextVisible } from "./helpers/permissions";
+import {
+  checkNotificationExists,
+  dismissNotification,
+} from "./helpers/notification";
+import { validateOperation } from "./helpers/operations";
+
+let instance = randomInstanceName();
+
+test("instance operations are recognised on the Operations page", async ({
+  browser,
+  browserName,
+}) => {
+  // Setup
+  instance = `${browserName}-${instance}`;
+  const page = await browser.newPage();
```

Comment:
```suggestion
  page
}) => {
```
The framework is already giving us a new page for this test, no need to open a new page manually.